src/freenet/crypt/JceLoader.java

```diff
@@ -55,8 +68,20 @@ static private boolean checkUse(String prop, String def)
 		}
 		BouncyCastle = p;
 		// optional
+		if (checkUse("use.SunJCE")) {
+		    try{
+		        KeyGenerator kgen = KeyGenerator.getInstance("AES", "SunJCE");
+		        kgen.init(256);
+		    }
+		    catch(Throwable e) {
+		        final String msg = "Error with SunJCE. Unlimited policy file not installed.";
```

Comment:
It looks like this string was intended to also be printed to stdout / stderr? Otherwise I'd expect it to be within the logger call.

Longer-term I'm interested in Freenet moving to an existing logging framework and doing stdout/stderr with log level configuration instead of manually.